### DISCUSSION TITLE
Backing up a Draft schema in Nebula studio

### DISCUSSION THREAD

**Discussion by PabloPete** [vesoft-inc/nebula#5681] [Q&A]
I have a Nebula studio instance that is not running in a Docker container and have created a Draft Schema that several people are editing. Is it possible to back this up and restore it and not just the PNG image ?

**Comment by wey-gu** [comment]
This is not yet supported and I consider it as a valid requirement(same pain points here)!

**Comment by wey-gu** [comment]
Issue was created here: https://github.com/vesoft-inc/nebula-studio/issues/618
